src/Symfony/Component/Translation/Extractor/PhpStringTokenParser.php

```diff
@@ -45,6 +47,8 @@
  * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * @deprecated since Symfony 6.2
```

Comment:
this should be in the phpdoc of the class, not in the license header.

Reply:
Indeed, thanks